app/deepseg.cc

```diff
@@ -631,22 +661,35 @@ int main(int argc, char* argv[]) try {
 		// copy new frame to buffer
 		cap.retrieve(raw);
 		ti.retrns = timestamp();
+
+		if (raw.rows == 0 || raw.cols == 0) continue; // sanity check
+
+		if ( crop_region.height) {
+			raw((cv::Rect_<int>)crop_region).copyTo(raw);
+		}
```

Comment:
```suggestion
		if (crop_region.height) {
			raw(crop_region).copyTo(raw);
		}
```

Reply:
I have updates my cropping branch accordingly. Your suggestion is within my code

Reply:
corrected within my branch

Reply:
This is modified like the code for line 633